deepchem/feat/molecule_featurizers/grover_featurizer.py

```diff
@@ -153,26 +141,12 @@ def _make_mol_graph(self, mol: RDKitMol) -> GraphData:
                 f_bonds.append(f_atoms[a1] + f_bond)
                 f_bonds.append(f_atoms[a2] + f_bond)
 
-                b1 = n_bonds
```

Comment:
Why are we deleting these?

Reply:
These attributes can be extracted from `extract_grover_attributes` method.